packages/office-ui-fabric-react/src/components/FloatingPicker/PeoplePicker/examples/FloatingPeoplePicker.Basic.Example.tsx

```diff
@@ -10,128 +9,95 @@ import {
 import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
 import { people } from '@uifabric/example-data';
 
-export interface IPeoplePickerExampleState {
-  peopleList: IPersonaProps[];
-  searchValue: string;
-}
-
-export class FloatingPeoplePickerTypesExample extends React.Component<{}, IPeoplePickerExampleState> {
-  private _picker = React.createRef<IBaseFloatingPicker>();
-  private _inputElement: HTMLInputElement;
-
-  constructor(props: {}) {
-    super(props);
-
-    this.state = {
-      peopleList: people,
-      searchValue: '',
-    };
-  }
-
-  public render(): JSX.Element {
-    const suggestionProps: IBaseFloatingPickerSuggestionProps = {
-      footerItemsProps: [
-        {
-          renderItem: () => {
-            const picker = this._picker.current;
-            return <div>Showing {picker ? picker.suggestions.length : 0} results</div>;
-          },
-          shouldShow: () => {
-            const picker = this._picker.current;
-            return !!picker && picker.suggestions.length > 0;
-          },
-        },
-      ],
-    };
-
-    return (
-      <div>
-        <div style={{ width: 208 }} ref={this._setInputElementRef}>
-          <SearchBox
-            placeholder="Search for person"
-            onChange={this._onSearchChange}
-            value={this.state.searchValue}
-            onFocus={this._onFocus}
-          />
-        </div>
-        <FloatingPeoplePicker
-          suggestionsStore={new SuggestionsStore<IPersonaProps>()}
-          onResolveSuggestions={this._onFilterChanged}
-          getTextFromItem={this._getTextFromItem}
-          pickerSuggestionsProps={suggestionProps}
-          key="normal"
-          onRemoveSuggestion={this._onRemoveSuggestion}
-          onValidateInput={this._validateInput}
-          componentRef={this._picker}
-          onChange={this._onPickerChange}
-          inputElement={this._inputElement}
-          resolveDelay={300}
-        />
-      </div>
-    );
+let inputElement: HTMLInputElement;
```

Comment:
This should go inside the component body, not here (and move other things inside the component body as needed). Otherwise different instances of the component would interfere with each other and the element won't get garbage collected when the component unmounts.

Reply:
I suggested to him to useRef here.